cnv/cnv-attaching-vm-multiple-networks.adoc

```diff
@@ -20,9 +20,9 @@ include::modules/cnv-connecting-resource-bridge-network.adoc[leveloffset=+1]
 
 [NOTE]
 ====
-In the next section, you must define a vNIC that refers to the bridge
-network by selecting the NetworkAttachmentDefinition object you created in
-the previous section.
+When defining the vNIC in the next section, ensure that the *NETWORK* value is
```

Comment:
s/*NETWORK*/`NETWORK`

Reply:
@ahardin-rh Thanks! Does the syntax in cnv-vm-create-nic-web.adoc need to be fixed also?

Reply:
@ousleyp Hmm it looks like that procedure pertains to the web console and I interpreted *NAME*, *NETWORK*, and *MAC ADDRESS* as fields in the UI. If that's the case, the way it is there is correct. If NETWORK is a variable set in a file and not used in the context of the UI, then we would place it in backticks, per 
https://github.com/ahardin-rh/openshift-docs/blob/enterprise-4.1/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

Reply:
@ahardin-rh Ok, cool - NETWORK here refers to the UI field that will come up in the cnv-vm-create-nic-web.adoc module. I think that means I can keep it in *s in the assembly as well. Thanks for your help!!